bin/ham-lint-fix-lib.sh

```diff
@@ -0,0 +1,420 @@
+#!/bin/bash
+export HAM_LINT_ALL_FILENAME="_lint_all_files.txt"
```

Comment:
Took me a bit to realize what this `_lint_all_files.txt` file was, I think it should be documented somewhere what the file is and the expected format of the text file (even just in a comment here in this script). Something like

```bash
# The `_lint_all_files.txt` contains a line-separated list of linting directives for a module.
# A linting directive is of the form `{LANGUAGE} {RELATIVE/PATH/TO/SRC/DIR}`
```

Reply:
Noted, I'll document it if it survives the next PR. I'm considering just getting rid of this altogether and doing "per file" only coupled with just using _build.ham to list the files which is just more flexible and not yet another kind of file to worry about.